김현국/app/src/main/java/com/example/myapplication/presenter/view/detail/DetailViewModel.kt

```diff
@@ -0,0 +1,75 @@
+package com.example.myapplication.presenter.view.detail
+
+import androidx.lifecycle.SavedStateHandle
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.example.myapplication.domain.Results
+import com.example.myapplication.domain.usecase.GetUserFollowerUseCase
+import com.example.myapplication.domain.usecase.GetUserFollowingUseCase
+import com.example.myapplication.presenter.UiState
+import com.example.myapplication.presenter.mapper.mapperDomainDetailUserToPresenterModel
+import com.example.myapplication.presenter.model.PresenterOwner
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.launch
+
+/**
+ * @author 김현국
+ * @created 2022/05/01
+ */
+class DetailViewModel(
+    val savedStateHandle: SavedStateHandle,
+    private val getUserFollowerUseCase: GetUserFollowerUseCase,
+    private val getUserFollowingUseCase: GetUserFollowingUseCase
+
+) : ViewModel() {
+    val USER_NAME = "userName"
```

Comment:
변수 이름도 그렇고 사용되는 곳도 약간 상수처럼 사용되서
object로 정의되면 좋을 것 같아요.

```suggestion
   companion object {
       private val USER_NAME = "userName"
   }
```